base_export_async/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+Base Export Async
+=================
+
+Standard Export can be delayed in asynchronous jobs executed in the background and then send by email to the user.
```

Comment:
have a look at the readme fragment mechanism, so the standard OCA readme.rst is auto generated.
https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

Reply:
I'll take a look